packages/threat-composer/src/components/workspaces/WorkspaceInsights/components/MitigationStatus/index.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  colorChartsStatusInfo,
+  colorChartsStatusPositive,
+  colorChartsStatusHigh,
+  colorChartsStatusNeutral,
+  colorChartsStatusLow,
+} from '@cloudscape-design/design-tokens';
+import { useMemo, useCallback, FC } from 'react';
+import {
+  MITIGATION_STATUS_IDENTIFIED,
+  MITIGATION_STATUS_IN_PROGRESS,
+  MITIGATION_STATUS_RESOLVED,
+  MITIGATION_STATUS_RESOLVED_ABANDONED,
+  STATUS_NOT_SET,
+} from '../../../../../configs';
+import { useMitigationsContext } from '../../../../../contexts/MitigationsContext';
+import { mitigationStatus } from '../../../../../data';
+import DashboardNumber from '../../../../generic/DashboardNumber';
+import useMitigationListLinkClicked from '../../hooks/useMitigationListLinkClicked';
+import { WorkspaceInsightsProps } from '../../types';
+
+const MitigationStatus: FC<WorkspaceInsightsProps> = ({
+  onMitigationListView,
+}) => {
+  const { mitigationList } = useMitigationsContext();
+
+  const handleLinkClicked = useMitigationListLinkClicked(onMitigationListView);
+
+  const countNotSet = useMemo(() => mitigationList.filter(x => !x.status).length,
+    [mitigationList],
+  );
+
+  const countIdentified = useMemo(
+    () => mitigationList.filter(x => x.status === MITIGATION_STATUS_IDENTIFIED).length,
+    [mitigationList],
+  );
+
+  const countResolved = useMemo(
+    () => mitigationList.filter(x => x.status === MITIGATION_STATUS_RESOLVED).length,
+    [mitigationList],
+  );
+
+  const countInProgress = useMemo(
+    () => mitigationList.filter(x => x.status === MITIGATION_STATUS_IN_PROGRESS).length,
+    [mitigationList],
+  );
+
+  const countAbandoned = useMemo(
+    () => mitigationList.filter(x => x.status === MITIGATION_STATUS_RESOLVED_ABANDONED).length,
+    [mitigationList],
+  );
+
+  const handleAddMitigation = useCallback(() => {
+    onMitigationListView?.();
+  }, [mitigationList, onMitigationListView]);
+
+  return (
+    <ColumnLayout columns={1} borders="horizontal">
+      {!mitigationList.length ? (
+        <Box
+          margin="xxl"
+          padding="xxl"
+          color="text-body-secondary"
+          textAlign="center"
+        >
+          <b>No mitigations available</b>
+          <Box variant="p" color="text-body-secondary">
+            Start by adding a mitigation to this workspace
+          </Box>
+          <Button variant="primary" onClick={() => handleAddMitigation()}>Add a mitigation</Button>
+        </Box>
+      ) : (
+        <Box padding="s">
+          <PieChart
+            data={[
+              {
+                title: mitigationStatus.find(x => x.value === MITIGATION_STATUS_RESOLVED)?.label || 'Resolved',
+                value: countResolved,
+                color: colorChartsStatusPositive,
+              },
+              {
+                title: mitigationStatus.find(x => x.value === MITIGATION_STATUS_RESOLVED_ABANDONED)?.label || 'Abandoned',
+                value: countAbandoned,
```

Comment:
Upon reflection, 'Abandoned' carries a negative connotation - and that it is simply left behind or neglected. I propose we change to 'Will not action' which carries slightly less negative connotation in my opinion, and is clear that it's a _decision_:

`countWillNotAction`